README.md

```diff
@@ -194,6 +194,20 @@ With the [`ini_setting` module](https://forge.puppet.com/modules/puppetlabs/inif
 
 5. Verify your Puppet data is in Datadog by searching for `sources:puppet` in the [Event Stream][5].
 
+### NPM setup
+
+To enable the Datadog Agent Network Performance Monitoring (NPM) features follow these steps:
+
+1. (Windows only) If the Agent is already installed, uninstall it by passing `win_ensure => absent` to the main class.
+2. (Windows only) Pass the `windows_npm_install` option with value `true` to the `datadog::datadog_agent` class. Remove `win_ensure` if added on previous step.
+3. Use the `datadog_agent::system_probe` class to properly create the configuration file:
+
+```conf
+class { 'datadog_agent::system_probe':
+    enabled => true, 
+}
```

Comment:
It seems like, on Windows, at least with recent versions, `enabled => true` doesn't make the system-probe component start, and you need:
```
class { 'datadog_agent::system_probe':
    network_enabled => true, 
}
```
instead.

This is probably due to the fact that not setting `network_enabled => true` makes puppet write:
```
network_config:
  enabled: false
```
explicitly in the config.